portfolio/src/main/webapp/style.css

```diff
@@ -78,29 +78,28 @@ html {
   border-radius: 25px;
   overflow: hidden;
   font-family: Verdana, Geneva, Tahoma, sans-serif;
-  position:relative;
+  position: relative;
   -ms-transform: translateY(-50%);
```

Comment:
I think you can remove this property, it looks like it's deprecated according to `make validate`